apps/fluent-tester/src/TestComponents/Menu/MenuRefs.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ButtonV1 as Button } from '@fluentui/react-native';
+import { Menu, MenuItem, MenuTrigger, MenuPopover, MenuList } from '@fluentui-react-native/menu';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+
+export const MenuTriggerChildRef: React.FunctionComponent = () => {
+  const ref = React.useRef(null);
+  const onClick = React.useCallback(() => {
+    ref?.current?.focus();
+  }, [ref]);
+
+  return (
+    <Stack style={stackStyle}>
+      <Button onClick={onClick}>Click to focus</Button>
```

Comment:
nit: "Click to focus on Menu Trigger" I've seen this in other tests and have been confused if clicking on the button focuses on itself or some other control.